project.Makefile

```diff
@@ -466,6 +466,7 @@ local/mongo_as_unvalidated_nmdc_database.yaml:
 local/mongo_as_nmdc_database_rdf_safe.yaml: nmdc_schema/nmdc_schema_accepting_legacy_ids.yaml local/mongo_as_unvalidated_nmdc_database.yaml
 	date # 449.56 seconds on 2023-08-30 without functional_annotation_agg or metaproteomics_analysis_activity_set
 	time $(RUN) migration-recursion \
+		--migrator-name Migrator_from_9_1_to_9_2 \
```

Comment:
thanks @brynnz22

Reply:
Sounds good. I'll do this in the future. Thanks!